torchvision/datasets/pcam.py

```diff
@@ -0,0 +1,136 @@
+import os
+import os.path
+import pathlib
+from typing import Any, Callable, Optional, Tuple
+
+from PIL import Image
+
+from .utils import download_file_from_google_drive, extract_archive, verify_str_arg
+from .vision import VisionDataset
+
+
+class PCAM(VisionDataset):
+    """`PCAM Dataset   <https://github.com/basveeling/pcam>`_.
+
+    The PatchCamelyon dataset is a binray classification dataset with 327,680
+    color images (96 x 96px), extracted from histopathologic scans of lymph node
+    sections. Each image is annoted with a binary label indicating presence of
+    metastatic tissue.
+
+    This dataset requires the ``h5py`` package which you can install with ``pip install h5py``.
+
+     Args:
+         root (string): Root directory of the dataset.
+         split (string, optional): The dataset split, supports ``"trai"`` (default), ``"test"`` or ``"val"``.
```

Comment:
```suggestion
         split (string, optional): The dataset split, supports ``"train"`` (default), ``"test"`` or ``"val"``.
```

Plus, is there a reason to order `"val"` after `"test"`?